### ISSUE TITLE
Call of duty series, is this the only fix for the game crashing? / Серия Call of duty, это единственное исправление вылета игры?

### ISSUE THREAD

**Issue by MrVoyage** [Flowseal/zapret-discord-youtube#310]
[EN]How do I use Discord and my COD doesn't crash?

1. Launching discord.bat
2. I launch Discord and enter the voice channel
3. I close the Zapret window
4. Running service_remove.bat
5. I launch COD and play

[RU]Как я использую Discord и у меня не вылетает COD?

1. Запускаю discord.bat (logically, as administrator)
2. Запускаю Discord и захожу в голосовой канал
3. Закрываю окно Zapret
4. Запускаю service_remove.bat (logically, as administrator)
5. Запускаю COD и играю

**Comment by johnnyd-78** [comment]
Да, единственное. Вылет игры вызывает драйвер WinDivert, а обход без него работать не умеет. Так что или... или...

И это не фикс, а воркараунд ;)

**Comment by AlexOlmer** [comment]
Спасибо за оперативный ответ!

**Comment by Nik01oS** [comment]
За это может ли прилететь бан? Вроде как античит Ricochet ругается на WinDivert и на дает запустить игру

**Comment by EgorWeders** [comment]
> За это может ли прилететь бан? Вроде как античит Ricochet ругается на WinDivert и на дает запустить игру

Может
